web/src/app/broadcast.client.ts

```diff
@@ -0,0 +1,22 @@
+import { Client, XmtpEnv } from '@xmtp/xmtp-js';
+import { Wallet, getDefaultProvider } from 'ethers';
+import { broadcastConfigs } from './broadcast.config';
+
+export const getXmtpClient = async (address: string) => {
+  const { /*id,*/ name } = broadcastConfigs.find(({ address: a }) => a === address) || {};
```

Comment:
Do not use static broadcast addresses list. Moreover, here it's completely unnecessary.